processor/transformprocessor/README.md

```diff
@@ -307,6 +308,25 @@ Examples:
 
 - `convert_summary_sum_val_to_sum("cumulative", false)`
 
+### copy_metric
+
+`copy_metruc(Optional[name], Optional[description], Optional[unit])`
+
+The `copy_metruc` function copies the current metric, adding it to the end of the metric slice.
+
+`name` is an optional string. `description` is an optional string. `unit` is an optional string.
+
+The new metric will be exactly the same as the current metric.  You can use the optional parameters to set the new metric's name, description, and unit.
+
+**NOTE:** The new metric is appended to the end of the metric slice and therefore will be included in all the metric statements. It is a best practice to ALWAYS include a Where clause when copying a metric that WILL NOT match the new metric.
+
+Examples:
+
+- `copy_metric(name="http.request.status_code", unit="s") where name == "http.status_code`
+
+
+- `copy_metric(desc="new desc") where description == "old desc"`
```

Comment:
```suggestion
- `copy_metric(name="http.request.status_code", unit="s") where name == "http.status_code`
- `copy_metric(desc="new desc") where description == "old desc"`
```

Reply:
We purposefully keep these spaced to make the markdown better

Reply:
I see now, thanks. Looking at the lists in the readme again, it only takes a single newline between each item to space them out. We really just need a markdown linter/formatter though, so at the end of the day it doesn't matter much.